app/controllers/games_controller.rb

```diff
@@ -0,0 +1,45 @@
+class GamesController < ApplicationController
+
+  def index
+    @games = Game.available
+  end
+
+  def create
+    # For Development Purposes Only - Used to test Available list.
+    Game.create(white_player_id: 1)
+
+    redirect_to games_path
```

Comment:
If you save the game in a variable, then you can redirect directly to its show page here:

```ruby
game = Game.create(...)

redirect_to game
```